src/main/java/frc/robot/RobotContainer.java

```diff
@@ -4,15 +4,95 @@
 import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+import frc.robot.generated.TunerConstants;
 
 public class RobotContainer {
-    public RobotContainer() {
-        configureBindings();
-    }
+    private double MaxSpeed = 6; // 6 meters per second desired top speed
+    private double MaxAngularRate = 1.5 * Math.PI; // 3/4 of a rotation per second max angular velocity
+
+    private final SlewRateLimiter filterY = new SlewRateLimiter(3);
+    private final SlewRateLimiter filterX = new SlewRateLimiter(3);
+
+    /* Setting up bindings for necessary control of the swerve drive platform */
+    private final CommandXboxController joystick = new CommandXboxController(0); // My joystick
+    private final CommandSwerveDrivetrain drivetrain = TunerConstants.DriveTrain; // My drivetrain
+
+    private final SwerveRequest.FieldCentric drive = new SwerveRequest.FieldCentric()
+            .withDeadband(MaxSpeed * 0.1).withRotationalDeadband(MaxAngularRate * 0.1) // Add a 10% deadband
+            .withDriveRequestType(DriveRequestType.OpenLoopVoltage); // I want field-centric
+                                                                     // driving in open loop
+
+    private final SwerveRequest.RobotCentric robotCentric = new SwerveRequest.RobotCentric()
+            .withDeadband(MaxSpeed * 0.1).withRotationalDeadband(MaxAngularRate * 0.1)
+            .withDriveRequestType(DriveRequestType.OpenLoopVoltage);
+    private final SwerveRequest.SwerveDriveBrake brake = new SwerveRequest.SwerveDriveBrake();
+    private final SwerveRequest.FieldCentricFacingAngle snapToAngle = new SwerveRequest.FieldCentricFacingAngle()
+            .withDeadband(MaxSpeed * 0.1).withDriveRequestType(DriveRequestType.OpenLoopVoltage);
+    private final PhoenixPIDController headingController = snapToAngle.HeadingController;
+
+    private final Telemetry logger = new Telemetry(MaxSpeed);
 
     private void configureBindings() {
+        drivetrain.setDefaultCommand( // Drivetrain will execute this command periodically
+                drivetrain.applyRequest(() -> drive.withVelocityX(filterX.calculate(joystick.getLeftX() * MaxSpeed))// Drive
+                                                                                                                    // Forward
+                                                                                                                    // with
+                                                                                                                    // negative
+                                                                                                                    // Y
+                        .withVelocityY(-filterY.calculate(joystick.getLeftY() * MaxSpeed)) // Drive left with negative X
+                                                                                           // (left)
```

Comment:
The above is incorrect. We should be feeding the value from the Y-axis of the joystick into the X velocity, and vice versa. The comments are there to prevent someone from doing what we did, which was think that was a mistake and send X to X and Y to Y. We should probably move the comment to its own line, too, because of what the formatter does.